### ISSUE TITLE
Feature request: Make font files configurable

### ISSUE THREAD

**Issue by maikschneider** [maikschneider/bw_captcha#4]
The [CaptchaBuilder](https://github.com/Gregwar/Captcha/blob/master/src/Gregwar/Captcha/CaptchaBuilder.php#L407) can configured to use other font files than the 4 .ttfs that are included. It would be nice to add a setting to use custom fonts.

**Comment by schloram** [comment]
Hi @maikschneider ,
the issue creator from Bitbucket here :) If you want, I could look into this.

**Comment by maikschneider** [comment]
Hey @schloram!

Yeah, that would be nice, just create a PR! :)

I think we could do something link this:

```
plugin.tx_bwcaptcha {
    settings {
        ...
        fonts = EXT:example/Resources/Private/Fonts/alternative1.ttf,1:fonts/alternative2.ttf,..
```

And add a new function that randomly returns one of the resolved file paths:

```
$font = CaptchaBuilderUtility::getRandomFontFromSettings($this->settings);
$builder->build((int)$this->settings['width'], (int)$this->settings['height'], $font);
```

**Comment by schloram** [comment]
@maikschneider Good idea to also include files in the storage folders. 👍 Didn't think of this.

Please have a look at the linked pull request.

**Comment by schloram** [comment]
@maikschneider I just figured in an older project (TYPO3 9) we are using this extension as well and this change would be neat to have there too. Do you think you could cherry pick this into v1? Or is this too much to ask? :)

**Comment by maikschneider** [comment]
Oha, okay. Yes, I'll create one last v1 release in the weekend :)

**Comment by maikschneider** [comment]
@schloram I just released [version 1.3.0](https://github.com/maikschneider/bw_captcha/releases/tag/v1.3.0)
